### DISCUSSION TITLE
Sound Engine Improvements

### DISCUSSION THREAD

**Discussion by ayskura** [nesbox/TIC-80#1871] [Ideas]
It would be great to have some sound improvements
1 import simple samples, maybe 8bit ogg format 
2 basic reverb, detune and filter on tracks
this would make the sound engine more versatile and easier to create great tunes

**Comment by atesin** [comment]
hi....   this was extensively discussed in #2162 , even @bztsrc wrote a couple tools for what you are saying..

... but i finally gave up when i undestood the "sound sample" is just **A SINGLE CYCLE** of 32 "input samples", played in a loop

... i mean, if your want a sound of lets say 500hz, your "sound sample" period (a single cycle actually) would be 1/500secs = 2ms ....  you cant do so much with a "sound sample" of 2ms, maybe just play with harmonics or so (either not too much with 32 input samples)

but in homepage there are some tool carts to "forge" sounds :)
